test/baselines/vis_Widget/package.json

```diff
@@ -32,8 +32,7 @@
     "www/",
     "io-package.json",
     "LICENSE",
-    "main.js",
-    "test-widget.js"
```

Comment:
🔍 found one!

Reply:
That one is outdated, but `main.js` shouldn't be there - I agree.